WooCommerce/Classes/ViewRelated/Products/ProductsListViewModel.swift

```diff
@@ -161,25 +156,7 @@ class ProductListViewModel {
     }
 }
 
-// MARK: - Blaze banner visibility
 extension ProductListViewModel {
-    /// Checks for Blaze eligibility and user defaults to show the banner if necessary.
-    ///
-    @MainActor
-    func updateBlazeBannerVisibility() async {
-        shouldShowBlazeBanner = await isBlazeBannerVisible()
-    }
-
-    private func isBlazeBannerVisible() async -> Bool {
-        async let isSiteEligible = blazeEligibilityChecker.isSiteEligible()
-        async let storeHasPublishedProducts = (try? checkIfStoreHasProducts(siteID: siteID, status: .published)) ?? false
-        async let storeHasAnyOrders = (try? checkIfStoreHasOrders(siteID: siteID)) ?? false
-        guard await(isSiteEligible, storeHasPublishedProducts, storeHasAnyOrders) == (true, true, false) else {
-            return false
-        }
-        return !userDefaults.hasDismissedBlazeBanner(for: siteID)
-    }
-
     @MainActor
     private func checkIfStoreHasProducts(siteID: Int64, status: ProductStatus? = nil) async throws -> Bool {
```

Comment:
We can remove these `checkIfStoreHasProducts` and `checkIfStoreHasOrders` methods as we no longer use them.

Reply:
This is also removed in https://github.com/woocommerce/woocommerce-ios/pull/10960/commits/33bd112d7e4a7abb2a869a95f57c6d622465ceb8